DependencyInjection/Configuration.php

```diff
@@ -816,6 +824,7 @@ private function addGelfSection(ArrayNodeDefinition $handerNode)
                         ->scalarNode('hostname')->end()
                         ->scalarNode('port')->defaultValue(12201)->end()
                         ->scalarNode('chunk_size')->defaultValue(1420)->end()
+                        ->scalarNode('encoder')->end()
```

Comment:
Perhaps an `enumNode` would be better here:

```suggestion
                        ->enumNode('encoder')->values(['json', 'compressed_json'])->end()
```